nemo/collections/asr/models/__init__.py

```diff
@@ -14,7 +14,11 @@
 
 from nemo.collections.asr.models.asr_model import ASRModel
 from nemo.collections.asr.models.audio_to_audio_model import AudioToAudioModel
-from nemo.collections.asr.models.classification_models import EncDecClassificationModel, EncDecFrameClassificationModel
+from nemo.collections.asr.models.classification_models import (
+    ClassificationTranscribeConfig,
```

Comment:
why not rename it to `ClassificationInferenceConfig` or `ClassificationInferConfig`?

Reply:
just noticed we have not added this to `label_models.py` , lets add there as well.

Reply:
Renamed it